examples/common_v2/definitions/example_materialized_view.sqlx

```diff
@@ -0,0 +1,3 @@
+config {type: "view", materialized: true}
```

Comment:
supernit: git complaining that this and other new files don't have trailing newlines